yarn-project/prover-client/src/orchestrator/proving-state.ts

```diff
@@ -143,6 +143,11 @@ export class ProvingState {
     return this.txs;
   }
 
+  /** Returns the block number as an epoch number. Used for prioritizing proof requests. */
+  public get epochNumber(): number {
+    return this.globalVariables.blockNumber.toNumber();
+  }
```

Comment:
I'm a bit confused by the terminology, IIUC an epoch = N slots, 1 slot = 0 or 1 blocks, so the epoch number should be the same for N consecutive blocks? Is this just temporary until we have the epoch number in global variables?

Reply:
This is just temporary until we have epoch number in global vars